dev/build.image/build.gradle

```diff
@@ -383,6 +388,28 @@ if (isAutomatedBuild && !isIFIXBuild) {
             }
         }
     }
+    
+    
+    task packageOpenLibertyWebProfile9(type: PackageLibertyWithFeatures) {
+        dependsOn parent.subprojects.assemble
+        dependsOn ':com.ibm.websphere.appserver.features:publishFeatureResources'
+        withFeatures this.&webProfile9Features
+        packageServerConflict = true
+        outputTo packageDir
+        doLast {
+            ["$packageDir/wlp/NOTICES", "$packageDir/wlp/README.TXT", "$packageDir/wlp/lib/versions/openliberty.properties"].each { path ->
+                replaceBetaVersionAndEdition(path, bnd.libertyRelease, libertyBeta)
+            }
+            copy {
+                from "$packageDir/wlp/templates/servers/webProfile9/server.xml"
+                into "$packageDir/wlp/templates/servers/defaultServer"
+            }
+            copy {
+                from "$packageDir/wlp/templates/clients/jakartaeeClient9/client.xml"
```

Comment:
I find it strange to have a defaultClient with webProfile, but we have it in javaee 8 as well.  I suppose if you install jakartaee-9.0 on top with the feature utility you will need it.